src/ui/.gitignore

```diff
@@ -1,6 +1,6 @@
 .vscode
 .DS_Store
 .env
-node_modules
-
-generated_reports
+.api
+.database
+node_modules
```

Comment:
Same as above, seems that `.git`, `docs/` and `dist/` don't get generated

Reply:
Good spot, have removed.